### PR TITLE
front: rename catenary to electrification

### LINKED ISSUE #6115: Rename catenary to electrification
## Description
In the English version of the editor the tool to edit elctrification is called "catenary tool" and in the list of layers we have catenaries once again. Despite this, in the form the object is called "electrification". In the warning list there is "overlapping of catenaries".

## AC

- [ ] Catenary are renamed everywhere to electrification

### PR DESCRIPTION
**Pull request by Akctarus**
close #6115

### CODE REVIEW

**Review by shenriotpro — APPROVED** [review]
OK for tests/

**Review by eckter — APPROVED** [review]

**Review by Wadjetz — APPROVED** [review]

File: front/public/locales/en/translation.json [hunk 1]
```diff
@@ -174,7 +174,7 @@
     },
     "layers": {
       "buffer_stops": "Buffer stops",
-      "catenaries": "Catenaries",
+      "electrifications": "electrifications",
```
Comment (SharglutDev) [inline front/public/locales/en/translation.json]:
```suggestion
      "electrifications": "Electrifications",
```

**Review by SharglutDev — CHANGES_REQUESTED** [review]
Thank you for the PR. Tested the front part, only one comment :)

File: front/public/locales/en/translation.json (hunk 1 above)
Comment (Akctarus) [reply front/public/locales/en/translation.json]: Ty ! fixed :)

**Review by SharglutDev — APPROVED** [review]

**Review by flomonster — APPROVED** [review]
LGTM (review and tested)

**Review by Tguisnet — APPROVED** [review]

**Pull request merged**